catalyst/utils/torch.py

```diff
@@ -638,6 +638,12 @@ def load_checkpoint(path: str):
     return checkpoint
 
 
+def soft_update(target: nn.Module, source: nn.Module, tau: float) -> None:
+    """Updates the `target` data with the `source` one smoothing by ``tau`` (inplace operation)."""
```

Comment:
we need a proper docs here

Reply:
not one-liner ;)

Reply:
[Done](https://github.com/catalyst-team/catalyst/pull/1305/commits/d17a9a584e138d2c2da3a57194a076849e7c1ba2)?